dashboard/app/controllers/ai_tutor_interactions_controller.rb

```diff
@@ -62,4 +62,14 @@ def find_project_and_version_id(level_id, script_id)
       version_id: version_id,
     }
   end
+
+  # GET /ai_tutor_interactions
+  def index
+    return render(status: :forbidden, json: {error: 'This user does not have access to AI Tutor chat messages'}) unless current_user.can_view_student_ai_chat_messages?
```

Comment:
I'm always a little shaky on these slightly magical things, but doesn't the cancan rule in `ability.rb` already cover this?

Reply:
One might guess that is the case, but my new test `'index returns forbidden for teacher who can not view chats'` fails without this line. It might have to do with how I've stubbed `can_view_student_ai_chat_messages?`. Regardless seems like a little extra caution is ok